5.3.1 release notes (#264)
hazelcast/management-center-docs: docs/modules/release-notes/pages/5-3-1.adoc

= 5.3.1 Release Notes
:description: These release notes list any new features, enhancements, and fixes that were made between version 5.3.0 and 5.3.1 of Hazelcast Management Center (MC).

{description}

[[enh-531]]
== Enhancements

* **SQL Browser:**
** Added graceful handling for SQL Browser websocket buffer overflow. [MC-2092]
** Added an ability to resize SQL Browser left panel. [MC-2091]
** Added an ability to select SQL Browser results with a right mouse click. [MC-1938]
** Esc or clicking outside the Connector wizard now closes it. [MC-2022]
* Changed login time in the top right corner to 24 hour clock. [MC-2122]
* Added margins to Client tables. [MC-2110]

[[fixes-531]]
== Fixes

* **SQL Browser:**
** Fixed documentation link in SQL Browser. [MC-2183]
** Fixed SQL Browser results scroll jump after the first 1000 rows. [MC-2182]
** Fixed SQL Browser left panel message. [MC-2175]
** Fixed SQL Browser column truncation. [MC-2106]
* Fixed false-positive HealthCheck warnings during the cluster restart. [MC-1902]

> * Fixed various penetration testing findings. [MC-2061]

Reviewer: Not sure if this entry should be included in this release notes document.

Author: ok, removed

> * MC no longer allows running WAN Sync dry run if Merkle tree is not enabled. [MC-2145]

Reviewer 2: ```suggestion
* MC no longer allows running WAN Sync check if Merkle tree is not enabled on the publisher. [MC-2145]
```

> * MC no longer overrides WAN replication config params with default values. [MC-1461]

Reviewer: ... Replication configuration parameters with the ...

> * Removed confusing info about user creation in logs. [MC-1766]

Reviewer: Removed inaccurate information related to the user creation in the logs. ...

> * Fixed various Scripting-related issues. [MC-1543][MC-1530]

Reviewer: Fixed various scripting related issues including the saved scripts not being executed ...
